extensions/blocks/calendly/calendly.php

```diff
@@ -64,6 +64,7 @@ function load_assets( $attr, $content ) {
 	$classes                 = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr, array( 'calendly-style-' . $style ) );
 	$block_id                = wp_unique_id( 'calendly-block-' );
 	$is_amp_request          = class_exists( 'Jetpack_AMP_Support' ) && \Jetpack_AMP_Support::is_amp_request();
+	$is_p2_site              = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) && true === get_blog_option( get_current_blog_id(), 'o2_options' )['o2_enabled'];
```

Comment:
If P2 is eventually available on self-hosted sites, it may be best not to gate things to WordPress.com. Could we limit it to only checking for P2, or to not gate at all, and look for a specific dom element in the existing Calendly JS?

Reply:
> If P2 is eventually available on self-hosted sites, it may be best not to gate things to WordPress.com. Could we limit it to only checking for P2, or to not gate at all, and look for a specific dom element in the existing Calendly JS?

Thanks for pointing this out, I've been a bit confused about the future of P2. I changed the logic to check for jQuery and o2 instead of gating.